contributing/documentation/updating_the_class_reference.rst

```diff
@@ -109,4 +109,8 @@ features. You can then check what changed using the ``git diff`` command.
 Please only include changes that are relevant to your work on the API in your commits.
 You can discard changes in other XML files using ``git checkout``, but consider reporting
 if you notice unrelated files being updated. Ideally, running this command should only
-bring up the changes that you yourself have made.
+bring up the changes that you yourself have made. 
+
+The newly generated XML will not have content or description and must be manually added. 
+This will not be removed from subsequent runs of the doctool. See 
```

Comment:
Unnecessary note IMO, it should be assumed

Reply:
With the line before I maybe agree. Overall this was unintuitive for me coming from other tools that autogenerate docs/etc so I wanted to be explicit. Open to changing it though

Reply:
I'd say it's something people will notice, and it's natural it won't erase everything when you regenerate it IMO, it goes without saying in my opinion, you don't assume the content is wiped when it doesn't come from anywhere else

Reply:
Yeah I think you are right, especially given the previous line saying you must manually add something. Will rework